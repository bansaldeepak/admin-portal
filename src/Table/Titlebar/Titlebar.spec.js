import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { shallowToJson } from "enzyme-to-json";
import Titlebar from "./Titlebar";

configure({ adapter: new Adapter() });

describe("Titlebar", () => {
  it("Render Titlebar with default options", () => {
    const component = shallow(<Titlebar />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
