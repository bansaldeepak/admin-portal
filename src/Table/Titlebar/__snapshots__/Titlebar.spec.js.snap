// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titlebar Render Titlebar with default options 1`] = `
<WithStyles(ForwardRef(Toolbar))
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Grid))
    className="makeStyles-container-7"
    container={true}
  >
    <WithStyles(ForwardRef(Grid))
      className="makeStyles-title-5"
      item={true}
    >
      <WithStyles(ForwardRef(Typography))
        id="tableTitle"
        variant="h6"
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      className="makeStyles-actions-4"
      item={true}
    />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Toolbar))>
`;
